modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Blogs/BlogAppService.cs

```diff
@@ -32,6 +33,13 @@ public async Task<BlogDto> GetByShortNameAsync(string shortName)
 
             if (blog == null)
             {
+                FileInfo fi = new FileInfo(shortName);
+
+                if (!string.IsNullOrEmpty(fi.Extension))
+                {
+                    return null;
```

Comment:
Did you check if it is OK to return null from this method (check all usages)? We don't want "object reference not set..." exceptions.